### DISCUSSION TITLE
Pot | 插件列表

### DISCUSSION THREAD

**Discussion by giscus** [pot-app/pot-desktop#810] [General]
# Pot | 插件列表

跨平台划词翻译和OCR软件

https://pot-app.com/plugin.html

**Comment by CYM-YYds** [comment]
为什么Rapid与Paddle都用不了？
使用就显示：LoadLibraryExW { source: Os { code: 193, kind: Uncategorized, message: "%1 不是有效的 Win32 应用程序。" } }

**Reply by ferstar** [reply]
用我糊的这个插件：https://github.com/ferstar/pot-app-recognize-plugin-rapid-py

**Comment by LinsanLiu** [comment]
想设置欧陆词典生词本同步，但是找不到这个插件了，只有有道和扇贝插件可以下载

**Comment by WINGnfly** [comment]
Hi, I can't find the .potext file on the plugin's GitHub page. How do I use it correctly?

**Reply by zeta987** [reply]
‌‌The .potext files are in releases. If you can't find them and only see .zip-like files, the plugin author may not have updated to the latest version yet. You can submit an issue requesting the plugin author to update it.

**Reply by MinjieY** [reply]
click the template "generated from pot-app/pot-app-collection-plugin-template" and odify the file name and configure the file name according to the template guide.

**Reply by MinjieY** [reply]
template link:  https://github.com/pot-app/pot-app-collection-plugin-template

**Comment by elarbor** [comment]
[插件下载地址](https://pot-app.com/plugin.html#%E7%BF%BB%E8%AF%91)该放到界面上 
![image](https://github.com/user-attachments/assets/5dd8c7fe-00bf-4880-8df8-ecebd4a84b7b)

**Reply by Kelly0701ZJY** [reply]
把他放大页面,就会出现保存按钮

**Reply by elarbor** [reply]
![image](https://github.com/user-attachments/assets/9ec826f2-8e95-4feb-aa15-c213c18360e9)

**Reply by Fristline** [reply]
问题解决了吗，用的什么办法？

**Comment by JerryYang-30** [comment]
使用翻译插件免配置的原理是什么呢（

**Reply by zeta987** [reply]
翻譯平台的網頁通常都會發送請求到服務器，在翻譯平台的網頁上上先按 F12 打開 `開發人員工具` ，然後捕獲請求的 URL、headers 和 body ，然後在插件上使用這些參數去請求翻譯平台的後端

比如說這個 [MOJi辞書插件](https://github.com/TechDecryptor/pot-app-translate-plugin-moji_dict/blob/main/main.js)：
通過向MOJi辞書的 API 鏈接發送 POST 請求查詢單詞。請求體 (body) 中包含一個 `functions` 數組指定了名為 `search-all` 的操作，參數包括要查詢的文本 `text` 和類型代碼 `types` 為 102（102 為單詞、106 為文法、103&121 為例句）。

1. 首先檢查 API 請求是否成功 `if (res.ok)` 。

2. 如果成功，它會深入解析返回的 JSON 數據結構 `res.data` ，確保包含所需的搜索結果。

3. 創建一個空的 `explanations` 數組來存儲處理後的結果。

4. 對每一組返回的內容創建一個新對象，包含空的 `trait`(Pot標題) 、`explains`(Pot內文) 與 `title`(MOJi標題) 和 `excerpt` (MOJi內文)，將這些東西保存到 `explanations` 數組中。

5. 對每一組返回的內容（單詞、文法、例句）進行處理：
   - 創建新對象，包含 `trait`（Pot 標題）和 `explains`（Pot 內文）。
   - 將 MOJi 辞書返回的 `title` (MOJi標題) 和 `excerpt` (MOJi內文) 映射到新對象中。
   - 將新對象添加到 `explanations` 數組。

7. 返回包含 explanations 數組的結果對象。

大概是這樣將達到免配置的效果。

**Comment by Kol-Tang** [comment]
Google翻译最近用不了了， 一直loading。。。。。

**Reply by elarbor** [reply]
得科学上网😁
不过Steam++也可以
![image](https://github.com/user-attachments/assets/a031fe85-5a38-43a5-b529-693d07467acb)

**Comment by IdPll** [comment]
最近MOJI查词的插件用不了了...

**Reply by zeta987** [reply]
已提交[PR](https://github.com/TechDecryptor/pot-app-translate-plugin-moji_dict/pull/2)，之後如果插件有問題可以到各個作者的倉庫提 Issue 哦 
![image](https://github.com/user-attachments/assets/7a068746-8687-4d0e-a8d0-f6c71fab7eb1)

**Reply by Mayuri33-sern** [reply]
请问moji最近能正常使用了吗?我自己使用时还是无法使用moji……

**Reply by zeta987** [reply]
> 请问moji最近能正常使用了吗?我自己使用时还是无法使用moji……

你可以依照 [此討論](https://github.com/TechDecryptor/pot-app-translate-plugin-moji_dict/issues/1#issuecomment-2381109191) 暫時使用尚未合併的插件唷

**Reply by Mayuri33-sern** [reply]
刚刚没注意到 现在已经可以使用了，谢谢！

**Comment by song-hailong** [comment]
离线英汉词典插件 在没网的时候用不了呢？

**Reply by zeta987** [reply]
可以嘗試把語言偵測引擎改成本地
![image](https://github.com/user-attachments/assets/5f20b536-4646-4b93-b9c2-cff2915cc43b)

**Comment by oxauwae** [comment]
实在好用！

**Comment by nasyxx** [comment]
有没有人做个Claude的呢？

**Comment by 343830384** [comment]
我发现只要安装了外部插件, 就会导致服务器设置内所有的翻译工具列表无显示, 且失去翻译功能!!

**Comment by 5igm** [comment]
能不能提供一下tesseract插件的调用方式？软件自带的tesseract只能识别英文，我自己下载安装了一下tesseract，发现支持的语言非常多啊。而且还支持中文日文竖版等。我发现tesseract自身的功能其实很强大，但是不知道如何用外部插件的形式调用本地的tesseract。请问能不能说明一下？

**Reply by zeta987** [reply]
根據 POT 的 Tesseract 語言列表，確實支援多種語言：
https://github.com/pot-app/pot-desktop/blob/e78c9635d991b885b1bb543252249a20a062e8ff/src/services/recognize/tesseract/info.ts#L6-L30

實際測試阿拉伯文、韓文、日文、西班牙文都能正常運作，但必須手動選擇目標語言。結果如下：
![image](https://github.com/user-attachments/assets/f4f4eeb6-23ed-49cd-9d50-61dadebc0316)
![image](https://github.com/user-attachments/assets/ba0a9529-f4bc-47ae-a987-bd95ca0b855b)
![image](https://github.com/user-attachments/assets/a78126d5-c5fc-4f5f-bf7f-2fae318fcdd5)
![image](https://github.com/user-attachments/assets/ffc8bd57-f3ff-4138-be4f-53ce2a16b729)

**Reply by 5igm** [reply]
可不可以增加联合检测的功能？我知道tessercat本身支持同时检测多个语言。比如中文和英文。请问能否增加这种支持？

**Reply by zeta987** [reply]
> 可不可以增加联合检测的功能？我知道tessercat本身支持同时检测多个语言。比如中文和英文。请问能否增加这种支持？

你可能指的是自動語種檢測，這個要到 POT 的 [ISSUE](https://github.com/pot-app/pot-desktop/issues) 提看看
中文、日文等多語言識別是正常的，只是沒有自動偵測的功能
![image](https://github.com/user-attachments/assets/47419c4a-cfac-4e4c-ab2b-51f2c3299bc5)

**Comment by neuralcraftsman** [comment]
太好用了

**Comment by Tzulao55** [comment]
求开发deepseek的文字识别官方插件

**Comment by Strivy-ZSY** [comment]
如何上传自己的插件到该页面或通过官方审核🥺

[插件地址](https://github.com/Strivy-ZSY/pot-app-recognize-plugin-ollama)

**Reply by Strivy-ZSY** [reply]
好了，我知道了，谢谢！🥰

**Reply by elarbor** [reply]
![image](https://github.com/user-attachments/assets/17d6c49c-c374-4ef3-a470-b830d1822c77)

**Reply by Strivy-ZSY** [reply]
您好，此问题是您没有下载ollama
如果您要使用本地服务，请于[ollama官网](https://ollama.com/)先下载ollama
随后打开终端，输入 ollama --version检查是否安装成功
随后点击[ollama模型列表](https://ollama.com/search)搜索并下载符合您电脑硬件情况的模型
比如您要安装`qwen2.5:1.5b`模型，您可以在终端输入以下命令
```
ollama run qwen2.5:1.5b
```
安装成功后即可在POT中使用🥰
希望对您有所帮助🤗

**Reply by bushiliang** [reply]
你好，我也想上传自己的插件到该页面，请问能告诉我你是如何上传的吗？
分别是硅基流动的翻译与火山引擎的ocr
https://github.com/bushiliang/pot-app-translate-plugin-SiliconFlow
https://github.com/bushiliang/pot-app-ocr-plugin-volcengine

**Reply by Strivy-ZSY** [reply]
> 你好，我也想上传自己的插件到该页面，请问能告诉我你是如何上传的吗？ 分别是硅基流动的翻译与火山引擎的ocr https://github.com/bushiliang/pot-app-translate-plugin-SiliconFlow https://github.com/bushiliang/pot-app-ocr-plugin-volcengine

您可以在此项目提交PR，可以参考我之前的方式修改对应文件
https://github.com/pot-app/pot-app-plugin-list/pull/29
希望对您有帮助🌹

**Reply by bushiliang** [reply]
好的，谢谢你，我已经提交了PR，感谢

**Reply by AAAleaf** [reply]
直接胡一个符合openai格式的自定义api的插件，然后设置点参数，硅基流动的GLM-4.5-Airf翻译速度很快

**Comment by HuaiMin7** [comment]
截图翻译识别不到截图内容怎么解决呢

**Reply by Strivy-ZSY** [reply]
请问文本识别插件是什么？同时使用POT的文本识别功能看看所返回的红色内容是什么？

**Comment by motisheng** [comment]
为什么我下载的有道没有potext文件啊

**Reply by Strivy-ZSY** [reply]
有呀

https://github.com/TechDecryptor/pot-app-translate-plugin-youdao/releases/tag/2.0.0

![image](https://github.com/user-attachments/assets/2a6d3114-7ff1-4e26-b4c6-8f99f6726a57)

**Comment by motisheng** [comment]
现在就显示error怎么办




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "pot-app/pot-desktop"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2025年4月21日(星期一) 中午11:58
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [pot-app/pot-desktop] Pot | 插件列表 (Discussion #810)





 
有呀
 
https://github.com/TechDecryptor/pot-app-translate-plugin-youdao/releases/tag/2.0.0
 
image.png (view on web)

**Reply by Strivy-ZSY** [reply]
> 现在就显示error怎么办
> […](#)
> ------------------&nbsp;原始邮件&nbsp;------------------ 发件人: "pot-app/pot-desktop" ***@***.***&gt;; 发送时间:&nbsp;2025年4月21日(星期一) 中午11:58 ***@***.***&gt;; ***@***.******@***.***&gt;; 主题:&nbsp;Re: [pot-app/pot-desktop] Pot | 插件列表 (Discussion #810) 有呀 https://github.com/TechDecryptor/pot-app-translate-plugin-youdao/releases/tag/2.0.0 image.png (view on web) — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you commented.Message ID: ***@***.***&gt;

请提供截图，或者Error的详细信息🌼

**Comment by 1373301041** [comment]
文字识别界面的翻译按钮，点了没用

**Comment by MAGA2077** [comment]
请问如何把surface pen的侧边按钮或者底部按钮设置成快捷键？

**Comment by paopaotang001** [comment]
好像deepl和openaI的翻译插件无法使用了

**Comment by murenzzj** [comment]
有没有支持markdwon输出的办法呢

**Comment by XiaoHP14** [comment]
每次搜索自动加入生词本功能很需要，不然每次自己点击，就不丝滑了

**Comment by cj1367780602** [comment]
在ubuntu22.04下快捷键都不能使用，我在某个评论下看到可以通过Ubuntu的Settings->Keyboard->View and Customize Shortcuts->Custom Shortcuts中添加快捷键，我添加了划词翻译的快捷键之后，可以使用划词翻译功能了，但是截图ocr翻译仍无法使用

**Comment by zhengshikong-beep** [comment]
OCR可不可以尝试用电脑本身的QQ截图OCR或者微信同理，自带的不太好用

**Comment by wxb1202** [comment]
外部调用截图怎么操作，没看懂那个端口在哪里设置

**Comment by ZSQBFan** [comment]
我也糊了一个插件：[一个使用视觉理解大模型进行文字识别的插件](https://github.com/ZSQBFan/pot-app-recognize-plugin-llm-ocr)

**Comment by gulubao** [comment]
用cc写了个 paddle ocr vl 的插件. https://github.com/gulubao/pot-app-recognize-plugin-paddle-vl

**Comment by zhao004** [comment]
https://github.com/zhao004/pot-app-recognize-openai-third-parties
文字识别插件，支持对接符合 OpenAI 接口规范的各类模型服务商。

**Comment by MaolinSun32** [comment]
ALL-in-One! 做了个pot插件，将单词和翻译写入到obsidian仓库的指定文件，配合obsidian的第三方插件：space-repetition进行闪卡复习。项目链接：https://github.com/MaolinSun32/pot-app-collection-plugin-obsidian

**Comment by p0ssword** [comment]
![1776354359438.png](https://free.picui.cn/free/2026/04/16/69e10437ef974.png)
TTS接口好像挂了

**Comment by 12471220** [comment]
适配千问dashscope API的插件
https://github.com/12471220/pot-app-translate-plugin-qwen
